### ISSUE TITLE
i calculated custom anchors using alexyab mentioned code in readme file but IOU loss is different for different anchor number ?

### ISSUE THREAD

**Issue by akashAD98** [AlexeyAB/darknet#7666]
first i tried with anchor=9 then IOU-loss is =68.4
for anchor = 15  IOU loss= 71.48
for anchor = 30 IOU loss = 80.84

@AlexeyAB  can you please explain to me the meaning of this iou_loss ?? & which one should I prepare??
actually, I'm detecting very small objects so I have calculated custom anchors.

**Comment by ekesdf** [comment]
Choose the one with the lower IOU loss bc the model with the lowest IOU loss is more accurate

**Comment by ekesdf** [comment]
IOU is how much difference is between the label you made for the object and the label generated from your training model so if the difference is small IOU loss is small and when there is a big difference then the IOU loss will be bigger
